src/components/checkTodo.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+  <div class="status-container">
+    <img
+      class="unchecked"
+      v-if="!checked"
+      @click="toggleCheck"
+      src="../assets/unchecked.png"
+    />
+    <div class="checked-todo" v-else @click="toggleCheck">
+      <img class="circle" src="../assets/checked.png" />
+      <img class="vector" src="../assets/Vector.png" />
+    </div>
```

Comment:
You don't need to save the circle as an image you can create using a div and just change the border color and show the checkmark if it is checked 
Also please save all your assets as svg (occupies less space) try to avoid png and jpg/jpeg